cli.js

```diff
@@ -98,14 +102,17 @@ const footer = cli.flags.footer;
 
 const title = cli.flags.title === false ? false : cli.flags.title;
 
+const scale = cli.flags.scale !== undefined ? cli.flags.scale : 1;
+
 const [ width, height ] = cli.flags.minDimensions.split('x').map(function(d) {
   return parseInt(d, 10);
 });
-
+console.info(cli.flags)
```

Comment:
Could you remove the console.info log here?

Reply:
Removed